modules/sdk/project-templates/project-templates-portletmvc4spring-thymeleaf-portlet/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+
+<archetype-descriptor name="liferay-module-portletmvc4spring-portlet">
+	<fileSets>
+		<fileSet filtered="true" packaged="true">
+			<directory>src/main/java</directory>
+			<includes>
+				<include>**/*.java</include>
+			</includes>
+		</fileSet>
+		<fileSet filtered="true">
+			<directory>src/main/resources</directory>
+			<includes>
+				<include>**/*</include>
+			</includes>
+		</fileSet>
+		<fileSet filtered="true">
+			<directory>src/main/webapp</directory>
+			<excludes>
+				<exclude>*.png</exclude>
+				<exclude>**/*.html</exclude>
+			</excludes>
+		</fileSet>
+		<fileSet filtered="false">
+			<directory>src/main/webapp</directory>
+			<includes>
+				<include>*.png</include>
+				<include>**/*.html</include>
```

Comment:
Should we filter html pages just as we filter jsp pages in other templates?  What do you think?

Reply:
@gamerson Similar to JSPX, we should not filter the HTML. Filtering is only necessary if there is something to substitute.